### PR TITLE
feat: add new sas file to New File...

### LINKED ISSUE #811: Add New File... -> SAS file
**Is your feature request related to a problem? Please describe.**
Is it possible to add a *.sas to the text part of new file similar to how Python is there and SAS notebook has been added? Yes, I can add a text file and change to sas, but this would be more convenient.  And I do not want to use a notebook.

**Describe the solution you'd like**
add a new menu item to create sas file.

**Describe alternatives you've considered**

**Additional context**

**Environment**
SAS version

### PR DESCRIPTION
**Pull request by scnjdl**
**Summary**
resolves issue #811 
support create a new sas file from New File... in File menu

**Testing**
Click New File... in File menu
Select SAS File SAS from dropdown list
A untitled sas file opened in active window

### CODE REVIEW

**Comment by smorrisj** [comment]
Thanks for adding this! See comment above.

**Review by scottdover — COMMENTED** [review]
I think this looks good, but I'll leave @smorrisj to +1 since he left a comment

**Review by smorrisj — CHANGES_REQUESTED** [review]

File: client/src/node/extension.ts [hunk 1]
```diff
@@ -160,6 +160,11 @@ export function activate(context: ExtensionContext): void {
         ),
       );
     }),
+    commands.registerCommand("SAS.file.new", async () => {
+      await window.showTextDocument(
```
[the remark this answers is not in the document]
Comment (scnjdl) [reply client/src/node/extension.ts]: Done

**Review by scnwwu — APPROVED** [review]

**Comment by scnjdl** [comment]
@smorrisj I have updated code, can you approve? Thank you.

**Comment by tianliang657** [comment]
I've finished this new feature functional and I18N testing, looks good to me. Approving.

**Comment by tianliang657** [comment]
This new feature is working fine in the main branch.

**Pull request merged**